.github/workflows/dependencies.yml

```diff
@@ -46,8 +51,7 @@ jobs:
           lerna exec --parallel ncu -- --upgrade --reject='constructs,projen,typescript,${{ steps.list-packages.outputs.list }}' --target=minor
 
       - name: Upgrade lock file
-        # This will create a brand new `yarn.lock` file (this is more efficient than `yarn install && yarn upgrade`)
-        run: yarn install --force
```

Comment:
`install --force` didn't update any packages that hadn't explicitly changed in one the `package.json` files, i.e transitive dependencies. We need those upgrades as well.